installation-bundle/src/Database/Version430Update.php

```diff
@@ -25,7 +25,7 @@ class Version430Update extends AbstractMigration
     /**
      * @var Connection
      */
-    protected $connection;
+    private $connection;
```

Comment:
all of these _might_ be considered BC breaks because the property was `protected` before 🙈

Reply:
But luckily, all the classes are marked `@internal`, so we do not have to care. 🎉